### PR TITLE
CSOAR-1861: Add csoar cloud execution egress topic

### PR DESCRIPTION
**Pull request by ffelici-sumo**
Signed-off-by: Fabio Felici <ffelici@sumologic.com>

## Purpose of this pull request

This pull request mention the egress topic for cloud executions and how ask the ip address

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/cloud-soar/architecture.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ Cloud SOAR design and architecture follows Cybersecurity Industry standards and
 
 Cloud SOAR offers a patent-pending Automated Responder Knowledge (DF-ARK) module which applies machine learning to historical responses and threats. It recommends relevant Playbooks, paths of action to expedite the process, and responses to manage and mitigate similar incidents with better response time.
 
+Cloud Soar provides static egress for Cloud Executions. IP addresses can be entered into the allowlist. For a list of cloud soar addresses by region, contact [support](https://support.sumologic.com/hc/en-us).
```
Comment (jpipkin1) [inline docs/cloud-soar/architecture.md:31]:
```suggestion
Cloud Soar provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud Soar addresses by region, contact [support](https://support.sumologic.com/hc/en-us).
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
I have a few suggestions, @ffelici-sumo.

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@ffelici-sumo, when you are ready to publish the PR, click **Merge when ready**.

File: docs/cloud-soar/architecture.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ Cloud SOAR design and architecture follows Cybersecurity Industry standards and
 
 Cloud SOAR offers a patent-pending Automated Responder Knowledge (DF-ARK) module which applies machine learning to historical responses and threats. It recommends relevant Playbooks, paths of action to expedite the process, and responses to manage and mitigate similar incidents with better response time.
 
+Cloud Soar provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud Soar addresses by region, contact [support](https://support.sumologic.com/hc/en-us).
```
Comment (kimsauce) [inline docs/cloud-soar/architecture.md:31]:
```suggestion
Cloud SOAR provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud SOAR addresses by region, contact [Support](https://support.sumologic.com/hc/en-us).
```

File: docs/cloud-soar/architecture.md [hunk 3]
```diff
@@ -28,6 +28,8 @@ Cloud SOAR design and architecture follows Cybersecurity Industry standards and
 
 Cloud SOAR offers a patent-pending Automated Responder Knowledge (DF-ARK) module which applies machine learning to historical responses and threats. It recommends relevant Playbooks, paths of action to expedite the process, and responses to manage and mitigate similar incidents with better response time.
 
+Cloud SOAR provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud SOAR addresses by region, contact [Support](https://support.sumologic.com/hc/en-us).
```
Comment (jpipkin1) [inline docs/cloud-soar/architecture.md:31]:
```suggestion
Cloud SOAR provides static egress for Cloud executions. IP addresses can be entered into the allowlist. For a list of Cloud SOAR addresses by region, contact [Support](https://support.sumologic.com/support/s/).
```

**Pull request merged**